### ISSUE TITLE
Can you provide the pre-trained attributes prediction model to me?

### ISSUE THREAD

**Issue by wty-ustc** [genforce/interfacegan#72]
In your paper, you claimed This model is trained with multi-task losses to simultaneously predict smile, age, gender, eyeglasses, as well as the 5-point facial landmarks. Can you provide me with this model? Thank you very much!

**Comment by GaParmar** [comment]
I would also find this very useful

**Comment by ShenYujun** [comment]
Please try models [here](https://github.com/NVlabs/stylegan/blob/03563d18a0cf8d67d897cc61e44479267968716b/metrics/linear_separability.py#L21).

**Comment by forkbabu** [comment]
@ShenYujun these were for celebahq, anything for ffhq? Will both work the same ways since both have similar   brilliant facial capabilities?

**Comment by lizaitang** [comment]
> Please try models [here](https://github.com/NVlabs/stylegan/blob/03563d18a0cf8d67d897cc61e44479267968716b/metrics/linear_separability.py#L21).

Thanks, I used this classfiers to test the result, but the prediction logits seems different each time, have you met the same issue?

**Comment by federicoromeo** [comment]
Can anyone provide a code snippet to make those classifiers work?
I followed stylegan's and even stylespace's ([see my issue for details](https://github.com/betterze/StyleSpace/issues/32#issue-1464825890)) but i can't make those work!
